P/PPL/build_tarballs.jl

```diff
@@ -38,7 +38,7 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="GMP_jll", uuid="781609d7-10c4-51f6-84f2-b8444358ff6d"))
+    Dependency("GMP_jll"),
```

Comment:
Please, don't do these changes if you don't actually need to rebuild the package: this will trigger the registration of a new useless version.

In particular, I don't understand the need of this change, it's totally fine as it is now